ocrolib/__init__.py

```diff
@@ -1,7 +1,7 @@
 __all__ = [
```

Comment:
This list is outdated. You can do more cleanup here.

Reply:
Is that just used for `from ocrolib.xyz import *`? That idiom is not used in ocropy's code base itself. I would not want to break any tools relying on that but I would not add to the list, merely remove old modules.

Reply:
What you did is OK, it's what I meant.